### PR TITLE
Use /release/versions API endpoint

### PR DESCRIPTION
**Pull request by mickeyn**
This time without breaking it off the async api.

### CODE REVIEW

File: metacpan_web.conf.local [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# rename this file to metacpan::web.yml and put a ':' after 'name' if
```
Comment (oalders) [inline metacpan_web.conf.local]: Did you intentionally add the `.local` config to the repo?

Comment (mickeyn) [inline metacpan_web.conf.local]: oh, no :)

**Comment by mickeyn** [comment]
the `pod` test failing is odd... will have to investigate

**Comment by oalders** [comment]
push build is failing

**Comment by mickeyn** [comment]
yeah, that pod test failure is odd... couldn't yet find what's going on, other than the text having a tiny difference.

**Comment by mickeyn** [comment]
seems like the previous pod test failure was random and happened in different branches inc. master.

**Pull request merged**